src/main/webapp/dev/js/test/CommonTestFunctions.es6

```diff
@@ -33,20 +33,21 @@ function clearBootboxModalStub() {
     $('#test-bootbox-modal-stub').html('');
 }
 
+const bootboxStub = {
```

Comment:
@LiHaoTan Shouldn't the variable name indicate some relation to bootbox as well?